src/v2/Components/Pagination/CommercePagination.tsx

```diff
@@ -0,0 +1,31 @@
+import { Pagination } from "./index"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export const CommercePaginationFragmentContainer = createFragmentContainer(
```

Comment:
I need to circle back and transform the stitched schema to avoid this duplicated fragment container. Works for the time being.